src/components/responsive-flyout/ResponsiveFlyout.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import { Flyout, Overlay } from '../flyout';
+import ScrollWrapper from '../scroll-wrapper';
+import type { FlyoutProps } from '../flyout/Flyout';
+import OverlayHeader from './OverlayHeader';
```

Comment:
Let's make sure to sort the imports. (I get this comment on all PRs. I hate manually sorting our imports. We should really make a push to move EUA to eslint import sorting. That way, we can rely on eslint to do our busy work for us.)

Reply:
I can configure Intellij to do it. I didn't get any errors on that so this was an oversight. Must not be configured, even though linting is already enabled.